pkg/cable/wireguard/driver.go

```diff
@@ -291,6 +299,15 @@ func (w *wireguard) DisconnectFromEndpoint(remoteEndpoint types.SubmarinerEndpoi
 }
 
 func (w *wireguard) GetActiveConnections(clusterID string) ([]string, error) {
+
+	//klog.V(log.DEBUG).Infof("WG STATUS for cluster %s", clusterID)
```

Comment:
I think it's better not to keep commented code in the codebase, we can always have this PR for reference if you want to look at this code later for implementation.

Does it work without the GetActiveConnections?, I guess your driver is already idempotent, we could do the same in the ipsec versions and remove that method

Reply:
Removed. 
WG cable does not use GetActiveConnections.